example/Example.hs

```diff
@@ -111,6 +111,7 @@ main = do
         runSQL_ $ "INSERT INTO consumers_example_jobs "
           <> "(run_at, finished_at, reserved_by, attempts, message) "
           <> "VALUES (NOW(), NULL, NULL, 0, 'hello')"
+        runSQL_ "SELECT pg_notify('consumers_example_chan', '')"
```

Comment:
There's a `notify` function in `hpqtypes`, please use it instead.

Reply:
Done. :+1: